frontend/src/screens/Me.js

```diff
@@ -157,7 +158,8 @@ export default function Me() {
         fetchJournals(null, null);
         getDateOverview(auth.token)
             .then(dates => {
-                // Emily's code here
+                const dateStrings = dates.map(d=>d.toDateString());
+                  setValidDates(dateStrings);
```

Comment:
please wrap this setState function using if(isMounted()) to avoid memory leak

Reply:
Done